src/runtime/containerd-shim-v2/create.go

```diff
@@ -259,3 +272,80 @@ func doMount(mounts []*containerd_types.Mount, rootfs string) error {
 	}
 	return nil
 }
+
+func configureNonRootHypervisor(runtimeConfig *oci.RuntimeConfig) error {
+	userName, err := createVmmUser()
+	if err != nil {
+		return err
+	}
+
+	u, err := user.Lookup(userName)
+	if err != nil {
+	    return err
+    }
+
+    uid, err := strconv.Atoi(u.Uid)
+    if err != nil {
+        return err
+    }
+    gid, err := strconv.Atoi(u.Gid)
+    if err != nil {
+        return err
+    }
+	runtimeConfig.HypervisorConfig.Uid = uint32(uid)
+	runtimeConfig.HypervisorConfig.Gid = uint32(gid)
```

Comment:
you have some problems with the indentation

Reply:
My bad. I forgot golang uses tab instead of space 😅